nubilanding/src/components/blog/form.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from './styles.module.scss';
+
+class Form extends React.Component {
+  state = { email: '' };
+  constructor(args) {
+    super(args)
+    this.state = {
+
+    }
+  }
```

Comment:
El state está definido 2 veces :thinking:
```suggestion
  state = { email: '' };
```